rtsf-at-checkout-event-simulator/tests/all_events_all_suspect.json

```diff
@@ -1,144 +1,5 @@
 {
     "checkout_events": [
-        {
-            "device": "RSP",
```

Comment:
I think these should be replaced with rfid-roi-events, rather than remove. i.e. the resulting event that the RSP controller would create from these.

Reply:
updated